### ISSUE TITLE
publish a new release?

### ISSUE THREAD

**Issue by legrostdg** [eskimor/servant-purescript#42]
f88dae371ff141f966b34d69b78908a70539975a fixes compilation with servant 0.18

**Comment by eskimor** [comment]
Yeah, it likely breaks the build with earlier versions though. Need to check that i don't make other's people life miserable. I will hopefully be able to look into this soon. If someone else could check, I am happy to make a release anytime.
